.github/dependabot.yml

```
@@ -9,4 +9,4 @@ updates:
     schedule:
       interval: "daily"
     ignore:
-      dependency-name: "org.codehaus.groovy:groovy-all"
+      - dependency-name: "org.codehaus.groovy:groovy-all"
```

Proposing to add commons-beanutils to the list as part of this fix.  See [PR-5124](https://github.com/jenkinsci/jenkins/pull/5124) and the recommendation from @basil 
```suggestion
      - dependency-name: "org.codehaus.groovy:groovy-all"
      - dependency-name: "commons-beanutils:commons-beanutils"
```